RLTest/env.py

```diff
@@ -179,6 +180,7 @@ def getEnvByName(self):
             'moduleArgs': self.moduleArgs,
             'useSlaves': self.useSlaves,
             'useAof': self.useAof,
+            'decode_responses': self.decode_responses,
```

Comment:
Convention, also you can just set it to be `defaultDecodeResponses`

Reply:
I think in this case we must follow the handling of other args like useSlaves, as the caller may be passing in this parameter.  It took a fair bit of fiddling to get this right and not regressing python2, while also working for python3, and I believe it is correct as-is in the PR.